test/quicksiler-config-client.test.js

```diff
@@ -39,4 +39,22 @@ describe('QuicksilverConfigClient', () => {
       client.should.have.property('qsConfigPath').which.is.not.empty();
     });
   });
+
+  // getAllSettings().
+  describe('getAllSettings', () => {
+    // Check getSettings method.
+    it('getAllSettings() should be exposed', () => {
+      getConfigClient().getAllSettings.should.be.a.Function();
+    });
+
+    // Get all settings.
+    it('should return all settings in mb_config.json', () => {
+      const client = getConfigClient();
+      const response = client.getAllSettings();
+      // Check response to be an object.
+      response.should.be.an.instanceOf(QuicksilverConfigClient);
```

Comment:
@sergii-tkachenko Thoughts on why "should" is undefined where it works fine in other related tests?

```
  ․․․․․

  4 passing (18ms)
  1 failing

  1) QuicksilverConfigClient getAllSettings should return all settings in mb_config.json:
     TypeError: Cannot read property 'should' of undefined
      at Context.<anonymous> (test/quicksiler-config-client.test.js:55:15)
```